lb.tf

```diff
@@ -1,5 +1,5 @@
 resource "aws_lb" "app_lb" {
-  name               = substr("lb-${var.cluster_name}-${var.app_name}", 0, min(length("lb-${var.cluster_name}-${var.app_name}"), 32))
+  name               = trimsuffix(substr("lb-${var.cluster_name}-${var.app_name}", 0, min(length("lb-${var.cluster_name}-${var.app_name}"), 32)), "-")
```

Comment:
Ya know... It might be worth making `${var.cluster_name}-${var.app_name}` into a local since we're using it in a bunch of places. Not really the scope of this PR exactly but something to consider.